src/module.c

```diff
@@ -413,7 +413,8 @@ int DropIndexCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
       RMUtil_StringEqualsCaseC(argv[0], "_FT.DROP")) {
     RedisModule_Replicate(ctx, RS_DROP_IF_X_CMD, "v", argv + 1, argc - 1);
   } else {
-    RedisModule_Replicate(ctx, RS_DROP_INDEX_IF_X_CMD, "v", argv + 1, argc - 1);
+    // Remove DD if CRDT. Documents were deleted with RM_Call.
+    RedisModule_Replicate(ctx, RS_DROP_INDEX_IF_X_CMD, "v", argv + 1, argc - 1 - (!!isCrdt & delDocs));
```

Comment:
&&

Reply:
Will it equal `1`? We neglect the `DD` flag this way.

Reply:
Shouldn't it be: (!!(isCrdt && delDocs))?